src/gui/qgscolorbrewercolorrampdialog.h

```diff
@@ -19,6 +19,7 @@
 #include <QDialog>
 #include "qgspanelwidget.h"
 #include "qgscolorramp.h"
+#include "qgshelp.h"
```

Comment:
This should be moved to the .cpp file - only include stuff in headers if compilation fails otherwise.

Basically the aim of the game is to have as little includes as possible in the header, as each one slows done compilation.

Reply:
Ok i'll try to reduce the ˋ#includeˋs (impact)